examples/expressions/12_subsampling.py

```diff
@@ -0,0 +1,84 @@
+"""
+.. currentmodule:: skrub
+
+.. _example_subsampling:
+
+Subsampling for faster development
+==================================
+
+Here we show how to use :meth:`.skb.subsample_previews()
+<Expr.skb.subsample_previews>` to speed-up interactive creation of skrub
+expressions by subsampling the data when computing preview results.
+"""
+
+# %%
+from sklearn.ensemble import HistGradientBoostingRegressor
```

Comment:
In narrative examples, I prefer doing the import of a symbol in the cell where I use it.